topic0KeyPoint1/src/oop3_4/MyTime.java

```diff
@@ -0,0 +1,201 @@
+package oop3_4;
+
+public class MyTime {
+
+	private int hour;
+	private int minute;
+	private int second;
+
+	public MyTime() {
+		this.hour = 0;
+		this.minute = 0;
+		this.second = 0;
+	}
+
+	public MyTime(int hour, int minute, int second) {
+
+		try {
+			this.setTime(hour, minute, second);
+		} catch (IllegalArgumentException e0) {
+			System.out.println("Invalid hour minute or second");
+		} catch (Exception e1) {
+			System.out.println(e1.getMessage());
+		}
+	}
+
+	public void setTime(int hour, int minute, int second) throws IllegalArgumentException {
+
+		if (MyTime.isValidHour(hour))
```

Comment:
It is a good practice to always use { and }, no matter if the code block only has one line